### PR TITLE
docs(PPDSC-2290): Audio player documentation

### PR DESCRIPTION
**Pull request by Srinivasan-Ramasamy**
PPDSC-2290

**What**

1. Audio player documentation UI - http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2290-audio-player-documentation/components/audio-player/
2. Previous audio player page (existing users can still access the docs) - http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2290-audio-player-documentation/components/audio-player-old/

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [X] Updated relevant documentation

**I have tested manually:**
- [X] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [X] The screen reader reads and flows through the elements as expected.
- [X] There are no new errors in the browser console coming from this PR.
- [X] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

image

**Who should review this PR:**

**How to test:**

URL: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2290-audio-player-documentation/components/audio-player/

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2290-audio-player-documentation
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2290-audio-player-documentation/storybook

**Comment by Vanals** [comment]
Hey Srini it looks good to me! I was just testing the page and noticed the table we use is not responsive. I wonder if is something designers are aware and are testing?
If is something to fix, could you maybe raise a ticket? Does not look a good experience for certain breakpoints, in some is not readable at all.


https://user-images.githubusercontent.com/25549355/187889440-94048cc2-b7e9-44cb-96d4-9f8654cb60e7.mp4

**Review by Vanals — DISMISSED** [review]

**Comment by Srinivasan-Ramasamy** [comment]
> Hey Srini it looks good to me! I was just testing the page and noticed the table we use is not responsive. I wonder if is something designers are aware and are testing? If they agree and is something to fix, could you maybe raise a ticket? Does not look a good experience for certain breakpoints, in some is not readable at all.
> 
>  table-responsiviness.mp4

@Vanals Thank you for taking your time for the review. 

Thanks for capturing this scenarios :100: 

Current table structure whenever more column with data it showing horizontal scroll bar to look at the hidden column data(Without any blocker to the users). It's very difficult to do the responsive in table with td dom structure. Till 1024 breakpoint, we're hiding the side nav so we can able to the full table. Please suggest?

Here is the few screen shots:

image

image

File: site/pages/components/audio-player.tsx [hunk 1]
```diff
@@ -0,0 +1,1762 @@
...
+              description: 'The selected state of the SelectionListOption',
+            },
+            {
+              name: 'selectedIcon',
+              type: 'React.ReactNode',
+              default: 'IconFilledCheck',
+              description: 'Icon(s) of the SelectionListOption',
+            },
+          ],
+          overridesRows: [
```
Comment (mutebg) [inline site/pages/components/audio-player.tsx:1694]: IMO we should not show `Selection List` and `Selection List Option` components here. they are internal ones.
We should add only overrides but as part of Speed Control. Maybe we can ask  Mike when is back

Comment (Srinivasan-Ramasamy) [reply site/pages/components/audio-player.tsx:1694]: Sure @mutebg, we will check with Mike and update accordingly. Thanks for your review :+1:

**Comment by messmike** [comment]
> Hey Srini it looks good to me! I was just testing the page and noticed the table we use is not responsive. I wonder if is something designers are aware and are testing? If they agree and is something to fix, could you maybe raise a ticket? Does not look a good experience for certain breakpoints, in some is not readable at all.
> 
>  table-responsiviness.mp4

This is a good point @Vanals, we have a ticket in the backlog to look at this: https://nidigitalsolutions.jira.com/browse/PPNKD-531

File: site/pages/components/audio-player.tsx (hunk 1 above)
Comment (messmike) [reply site/pages/components/audio-player.tsx:1694]: Hi @Srinivasan-Ramasamy to Stoyan's point regarding the `Selection List` and `Selection List Option` components, I would say we should move the overrides that are currently attributed to both in those sections, under the `Playback Speed Control` section.

See the updates here in Confluence: [https://nidigitalsolutions.jira.com/wiki/spaces/NPP/pages/3933962254/Audio+Player+Component+-+Web+Documentation#Playback-Speed-Control.1]

Comment (messmike) [reply site/pages/components/audio-player.tsx:1694]: It looks like there is a missing inline message under the Playback Speed Control in the API section also to say:

> This component should extend the overrides of the Popover and Modal components.

Comment (Srinivasan-Ramasamy) [reply site/pages/components/audio-player.tsx]: Thanks @messmike for the confirmation :+1: 

I've updated the changes in same PR url. Please review it once. http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2290-audio-player-documentation/components/audio-player/

Inline message was kepted  inside the Overrides tab, Now I've moved to default visibile.

Comment (messmike) [reply site/pages/components/audio-player.tsx]: Thank you @Srinivasan-Ramasamy, this looks great, all looks good to me.

Comment (Srinivasan-Ramasamy) [reply site/pages/components/audio-player.tsx]: Thanks Mike :+1:

**Review by Vanals — APPROVED** [review]

**Review by baburay23 — APPROVED** [review]

**Pull request merged**